diesel_derives/src/identifiable.rs

```diff
@@ -44,5 +44,15 @@ pub fn derive(item: DeriveInput) -> TokenStream {
                 (#(&self.#field_name),*)
             }
         }
+
+        impl #ref_generics Identifiable for &'ident &'ident #struct_name #ty_generics
```

Comment:
Looks like this impl is not as general as it could be: the outer reference is unnecessarily constrained.
```suggestion
        impl #ref_generics Identifiable for &'_ &'ident #struct_name #ty_generics
```